Dockerfile-njs

```diff
@@ -0,0 +1,63 @@
+FROM behance/docker-base:4.0-ubuntu-20.04
```

Comment:
@kmashint : Please consider updating to docker-base:5.0 if possible especially if this is going to be a brand new image anyway

Reply:
You also mentioned Ubuntu 22.04, so should I align on this?
`FROM behance/docker-base:5.0-ubuntu-22.04`